### PR TITLE
Improve dev ux

### PR DESCRIPTION
**Pull request by PeterJCLaw**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -108,6 +108,16 @@ async def submit(request: Request) -> RedirectResponse:
 
 
 if __name__ == "__main__":
+    if not OUTPUT_DIR.exists():
+        print(f"Creating output directory at {OUTPUT_DIR}")
+        OUTPUT_DIR.mkdir(parents=True, exist_ok=True)
```
Comment (RealOrangeOne) [inline app.py:113]: Probably worth removing `exist_ok` here, as you're already checking whether it exists above.
```suggestion
        OUTPUT_DIR.mkdir(parents=True)
```

File: app.py [hunk 2]
```diff
@@ -108,6 +108,16 @@ async def submit(request: Request) -> RedirectResponse:
...
+
+    if not os.environ.get("COMMAND"):
+        print(
```
Comment (RealOrangeOne) [inline app.py:116]: I like this! Although thoughts on whether it should be a hard error?

File: app.py (hunk 1 above)
Comment (PeterJCLaw) [reply app.py:113]: We are checking above, however removing `exist_ok` introduces a race condition if something creates the directory in the meanwhile.

File: app.py (hunk 2 above)
Comment (PeterJCLaw) [reply app.py:116]: I had this as a warning so that you can still use the UI to view existing runs without needing to configure anything.

**Pull request merged**